single-p4_action-resistance_hub.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * Template Name: Resistance Hub Campaign
+ * The template for displaying Resistance Hub campaign actions.
+ *
+ * @package  WordPress
+ * @subpackage  Timber
+ * @since    Timber 0.1
+ */
+
+include get_template_directory() . '/single-p4_action.php';
+
+// Add this code here instead of adding inline
+?>
```

Comment:
Will check later if this approach is the best for now or need to enqueue css and/or javascript